### ISSUE TITLE
Option(and shortcut) to move selected text to child note

### ISSUE THREAD

**Issue by terafo** [zadam/trilium#927]

**Comment by zadam** [comment]
I think this is what you want: https://github.com/zadam/trilium/wiki/Text-editor#cut-selection-to-sub-note

**Comment by terafo** [comment]
Yes, it is. But it still would be great to be able to do it with mouse.

**Comment by zadam** [comment]
This gif shows how to do it with mouse, no? https://raw.githubusercontent.com/wiki/zadam/trilium/gifs/cut-to-subnote.gif

**Comment by terafo** [comment]
I mean something like this [https://send.firefox.com/download/c221c8afb826bff2/#jzuvGBKxzJNcO0Nt8FzNbQ](url).

**Comment by zadam** [comment]
Oh, ok, so more like a drag & drop a text from one note to another...
